releases.yaml

```diff
@@ -55,13 +55,15 @@ wsl:
 
 checksums:
   desktop:
+    "25.10": "32e30d72ae4798c633323a2684d94a11582bb03a6ab38d2b0d5ae5eabc5e577b *questing-desktop-amd64.iso"
```

Comment:
The checksum is right, but the image name is not. You need to craft an image name that is similar to the previous versions.
This applies to every image.